common/src/main/java/com/sequenceiq/cloudbreak/common/database/TargetMajorVersionToMajorVersionConverter.java

```diff
@@ -0,0 +1,11 @@
+package com.sequenceiq.cloudbreak.common.database;
+
+import org.springframework.stereotype.Component;
+
+@Component
+public class TargetMajorVersionToMajorVersionConverter {
```

Comment:
why is this converter necessary? it seems to do the same as `TargetMajorVersion#TargetMajorVersion`
I think only one of these should exist